en/resources/contributing.md

```diff
@@ -9,6 +9,8 @@ redirect_from: "/resources/community.html"
 
 # Contributing to Express
 
+### Looking to contribute to expressjs.com? Click [here](#expressjs-website-contributing).
+
```

Comment:
On line 269 it has this. I was think we should filter this out, or point it to the section below. 
```
### Website Issues
Open issues for the expressjs.com website in https://github.com/expressjs/expressjs.com.
```
![image](https://github.com/user-attachments/assets/c506a9cb-90ef-486e-b1a7-e465cf53938c)

This will be my final change to this PR. 
(Commenting was not allowed on the proper line in this file)

Reply:
Maybe it’s better to change it from the source point once this PR is merged.

Reply:
Except they will want to leave this on the Express repo as it is written there, but it doesn't make sense to have it on our page. 
It's a very small change though so we can leave it out if you'd prefer.